smoketest/simplenotification_india_test.go

```diff
@@ -1,15 +1,14 @@
 package smoketest
 
 import (
-	"github.com/target/goalert/smoketest/harness"
 	"testing"
+
+	"github.com/target/goalert/smoketest/harness"
 )
 
 // TestSimpleNotifications_India tests that SMS and Voice goes out for
 // 1 alert -> service -> esc -> step -> user. 2 rules (1 of each) immediately.
-//
-// Currently, country code '+222' is used as a negative test. If we support
-// 222 in the future, this test will need to be updated.
+// Since with a recent PR, GoAlert accepts all international phone numbers, the negative test for '+222' has been adjusted accordingly.
```

Comment:
Future readers of this won't have the context of what used to be here, how about we rename this to `NonUS` instead of just `India`, and we can test India and Mauritania here.

Or, alternatively, just add all 3 to `simplenotification` and get rid of this one.